ansible/roles/debops.dovecot/defaults/main.yml

```diff
@@ -74,16 +74,22 @@ dovecot_user_accounts: '{{ [ "deny", "ldap" ]
 # List of users for which mail acess is disabled.
 dovecot_deny_users: [ 'root' ]
 
+                                                                   # ]]]
+# .. envvar:: dovecot_vmail_home [[[
+#
+# The vmail home directory is a per-user directory where Dovecot can save user-specific files.
+# Dovecot's home directories have nothing to do with system users' home directories.
+# It's irrelevant if it's under /home/ or /var/mail/ or wherever.
+# For more information see: `Home Directories for Virtual Users <https://wiki.dovecot.org/VirtualUsers/Home>`_.
+dovecot_vmail_home: '/var/vmail/%d/%n'
+
                                                                    # ]]]
 # .. envvar:: dovecot_mail_location [[[
 #
 # Mailbox location. For mbox set something like ``mbox:~/mail:INBOX=/var/mail/%u``.
 # For more information about the supported format, check `Dovecot Mail
 # Location <https://doc.dovecot.org/configuration_manual/mail_location/>`_
-dovecot_mail_location: 'maildir:{{ "/var/vmail/%d/%n/mailbox"
-                                  if (dovecot__ldap_enabled|d(False))
-                                  else
-                                  "~/Maildir" }}'
+dovecot_mail_location: 'maildir:~/Maildir'
```

Comment:
Wrong rename of the mail direcoty to `mailbox`, thus reverting to default standard.